src/test/java/pl/gov/coi/pomocua/ads/BaseResourceTest.java

```diff
@@ -156,6 +156,42 @@ void shouldReturn404WhenSingleOfferNotFound() {
         }
     }
 
+    @Test
+    void shouldReturn404OnGetDeletedOffer() {
+        T created = postSampleOffer();
+        restTemplate.delete("/api/secure/" + getOfferSuffix() + "/{id}", getClazz(), created.id);
```

Comment:
It could be simplified to `restTemplate.delete("...url...")`.
Actually, the method used here is:
`public void delete(String url, Object... urlVariables)`
(from `TestRestTemplate` javadoc)
which makes no sense here, since we're not using URL variables in this endpoint